job-dsls/jobs/prod/prod_pr_jobs.groovy

```diff
@@ -0,0 +1,122 @@
+def configs = [:]
+
+configs['izpack'] = [
+        mavenParams: '',
+        pmeParams: '''
+         -DversionOverride=${IZPACK_VERSION}
+        '''
+]
+
+configs['installer-commons'] = [
+        mavenParams: '',
+        pmeParams: '''
+         -DversionOverride=${INSTALLER_COMMONS_VERSION}
+         -DizpackVersion=${IZPACK_VERSION}
+         -DgroovyScripts=file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/InstallerCommonsAlignment.groovy 
+        '''
+]
+
+configs['rhba-installers'] = [
+        mavenParams: '',
+        pmeParams: '''
+         -DversionOverride=${RHPAM_VERSION}
+         -DizpackVersion=${IZPACK_VERSION}
+         -DinstallerCommonsVersion=${INSTALLER_COMMONS_VERSION}
+         -DkieVersion=${KIE_VERSION}
+         -DgroovyScripts=file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/InstallerAlignment.groovy,file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/InstallerCommonsAlignment.groovy,file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/ForceComRedhatBaVersion.groovy,file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/ForceComRedhatBaVersionLast.groovy
+        '''
+]
+
+configs['bxms-patch-tools'] = [
+        mavenParams: '-Prhpam,rhdm,integration-tests',
+        pmeParams: '''
+         -DversionOverride=${RHPAM_VERSION} 
+         -DkieVersion=${KIE_VERSION} 
+         -DupdatableVersionRange=7.9.0.redhat-00002,7.9.1.redhat-00003,7.10.0.redhat-00004,7.10.1.redhat-00001
+         -DgroovyScripts=file:///${CONFIG_CHECKOUT_PATH}/rhba/nightly/BxmsPatchToolsPropertiesAlignment.groovy 
+        '''
+]
+
+def folderPath = "PROD/main/pullrequest"
+folder("PROD")
+folder("PROD/main")
+folder("PROD/main/pullrequest")
+
+def commands = this.getClass().getResource("job-scripts/prod_pr_jobs.jenkinsfile").text
+
+configs.each { repository, config ->
+
+    def branchName = repository == 'izpack' ? 'bxms-7.0' : 'main'
```

Comment:
@almope why don't you move branch to `configs`? :thinking:

Reply:
@Ginxo I wanted to keep the `configs` for the build parameters, both for PME or Maven in this case. But could move the `branch` there if you think it makes more sense to have it there.

Reply:
I think it's clearer to have it there in the config yes.

Reply:
Moved to the configs.